--- GIDs.orig	2015-08-10 14:15:55.676605064 +0000
+++ GIDs	2015-08-10 14:17:22.090597751 +0000
@@ -322,6 +322,7 @@
 rpkirtr:*:995:
 tarantool:*:996:
 bareos:*:997:
+graylog:*:998:
 ebnetd:*:999:
 nogroup:*:65533:
 nobody:*:65534:
--- UIDs.orig	2015-08-10 14:15:44.684602874 +0000
+++ UIDs	2015-08-10 14:16:44.699598711 +0000
@@ -330,5 +330,6 @@
 rpkirtr:*:995:995::0:0:RPKI router server:/nonexistent:/usr/sbin/nologin
 tarantool:*:996:996::0:0:Tarantool Daemon:/nonexistent:/usr/sbin/nologin
 bareos:*:997:997::0:0:Bareos Daemon:/var/db/bareos:/usr/sbin/nologin
+graylog:*:998:998::0:0:Graylog:/nonexistent:/usr/sbin/nologin
 ebnetd:*:999:999::0:0:EBNETD:/nonexistent:/usr/sbin/nologin
 nobody:*:65534:65534::0:0:Unprivileged user:/nonexistent:/usr/sbin/nologin
